src/shared/__stories__/drop-target.stories.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import styled from "react-emotion";
+import { storiesOf } from "@storybook/react";
+import { action } from "@storybook/addon-actions";
+
+import DropTarget from "../components/drop-target";
+import { ContainedButton } from "../components/buttons";
+
+const DropTargetContent = styled.div`
+  align-content: center;
+  align-items: center;
+  display: grid;
+  grid-gap: 20px;
+  justify-content: center;
+  justify-items: center;
+`;
+
+export const actions = {
+  onDrop: action("onDrop")
+};
+
+const stories = storiesOf("DropTarget", module);
+
+stories.add("default", () => (
```

Comment:
it'd be great if there were some feedback after I drag and drop in these stories. Can you add something like listing the file names below the element? One way we've accomplished this is to just write a component function that utilizes the `DropTarget` component you have here + uses hooks to show local state in a list or something.